### ISSUE TITLE
ACF compatibility inconsistency

### ISSUE THREAD

**Issue by shabnam611** [weDevsOfficial/wp-user-frontend#983]
Description: If we keep the ACF compatibility no, but keep the ACF plugin activated, the custom fields' data don't appear in the backend of custom post type post. 

How to Reproduce the Issue: 

1. First navigate through WP-admin > WPUF > Settings > General Settings > set ACF compatibility to no. Keep the ACF plugin installed and activated on your site.

2. Next create a custom post type post form from WPUF with some custom fields, submit the shortcode on a page. Visit the page as a user and then submit the post. Please insert data in all custom fields.

3. Visit the custom post type post from WP-admin area and check for custom fields. You will find no custom fields of WPUF custom post type post form. 

4. Next navigate through the plugins section and deactivate the ACF plugin. Then just visit the custom post type post we submitted in step 2. The custom fields will appear. 

Expected scenario: The custom field data should appear in the custom post type post in the backend even when the plugin is activated because the compatibility is set to no so ACF should not appear but the custom field data should appear. 

Video: https://www.loom.com/share/52e21d8c6aa54796bec3541a0dc39df7

**Comment by mmarj** [comment]
According to the integration between ACF & WPUF: Either you should sync all of your existing WPUF custom fields with the ACF custom fields, or you should use WPUF or ACF standalone, which is okay.

Let me explain the consequences a little more:

As we know, the custom fields meta/data appear within a meta_box of the associated post. And then, you have both ACF & WPUF plugins installed to meet the integration between these two plugins. So to meets the primary purpose of the integration with the post's custom fields meta, it will only show the ACF's custom meta_box on the backend area.

So, if you want to use WPUF with the ACF, then you should map & sync all of the existing custom fields with ACF (also enable compatible mode on the settings). Upon prioritizing, it will only show the ACF meta_box on the backend after data insertion, which will serve both of the ACF & WPUF's custom meta/data. And this is what happens on the system, so I'm not sure why you create this issue.

In short:
If you have enabled the ACF compatible mode and you have both ACF & WPUF plugin installed, then you have made the custom fields map & sync in between ACF & WPUF. After the user submits the data, it will only show the ACF's meta_box filed on the post backend with the associated mapped field.

**Noteworthy**: Also, the "ACF custom fields option" should be checked from the post backend area- http://prntscr.com/unw7lf to make it available.

So I would request you to please have a look once again & also let me know if I'm wrong! :)
Thanks.

**Comment by shabnam611** [comment]
When use keeps compatibility no, he doesn't want to use ACF with WPUF, probably he is using the plugin for any other purpose. In that case, he is unable to see the custom field data of WPUF in the backend of custom post. 

If the user says he doesn't want compatibility between ACF and WPUF, ACF should not work with WPUF anymore, and I assume that's why users can enable/disable compatibility between ACF & WPUF.

**Comment by hasinur1997** [comment]
@shabnam611 Couldn't reproduce the issue. Please take look https://www.loom.com/share/1c2d6d7f5ef943d2b12747e01725e980
